### PR TITLE
Add a test to cover stencil-only blits.

### PR DESCRIPTION
**Pull request by null77**
Also move the existing tests into the skip list. A bug in AMD makes
these tests hard to pass, to put them in 2.0.1 after which AMD might
be able to release a fixed driver.

We've reported this as a standalone repro to AMD, see:
https://github.com/null77/d3d11-driver-bugs/tree/master/StencilBlit

Hopefully this should be fixed soon. In the meantime working around this
issue is a bit tricky so I'd like to move it to a standalone test for 2.0.1, added
in this pull request.

@kenrussell @jdashg @zhenyao

### CODE REVIEW

File: sdk/tests/conformance2/rendering/blitframebuffer-stencil-only.html [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+var wtu = WebGLTestUtils;
+description("This test covers some edge cases of blitFramebuffer with stencil.");
+
+var gl = wtu.create3DContext("example", undefined, 2);
+
+var program, colorLoc;
+
+function checkPixel(color, expectedColor) {
+  var tolerance = 7;
```
Comment (zhenyao) [inline sdk/tests/conformance2/rendering/blitframebuffer-stencil-only.html]: nit: maybe less, say 3?

File: sdk/tests/conformance2/rendering/blitframebuffer-stencil-only.html [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+// Test based on dEQP-GLES3.functional.blit.depth_stencil.depth_24_stencil8_stencil_only
+function test_stencil_only_blit(format) {
+  var src = init_buffer(format);
+  var dest = init_buffer(format);
+
+  gl.bindFramebuffer(gl.FRAMEBUFFER, src.fbo);
+  gl.viewport(0, 0, 16, 16);
+
+    // Fill source with red, depth = 0.5, stencil = 7
```
Comment (zhenyao) [inline sdk/tests/conformance2/rendering/blitframebuffer-stencil-only.html]: nit: wrong indent

File: sdk/tests/conformance2/rendering/blitframebuffer-stencil-only.html [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+  gl.vertexAttribPointer(0, 3, gl.FLOAT, gl.FALSE, 0, 0);
+  gl.enableVertexAttribArray(0);
+  gl.drawArrays(gl.TRIANGLES, 0, 6);
+
+  wtu.glErrorShouldBe(gl, gl.NO_ERROR, "after drawQuad");
+}
+
+// Test based on dEQP-GLES3.functional.blit.depth_stencil.depth_24_stencil8_stencil_only
+function test_stencil_only_blit(format) {
+  var src = init_buffer(format);
```
Comment (zhenyao) [inline sdk/tests/conformance2/rendering/blitframebuffer-stencil-only.html]: nit: can we add a debug("...") here for the format being tested?

File: sdk/tests/conformance2/rendering/blitframebuffer-stencil-only.html (hunk 1 above)
Comment (kenrussell) [inline sdk/tests/conformance2/rendering/blitframebuffer-stencil-only.html]: Actually this function is unreferenced. Please just delete it. The test's already checking the color in the preferred way, using wtu.checkCanvasRect.

**Comment by null77** [comment]
Thanks for the reviews. I'll address these tomorrow morning and upload a new version.

File: sdk/tests/conformance2/rendering/blitframebuffer-stencil-only.html (hunk 1 above)
Comment (null77) [inline sdk/tests/conformance2/rendering/blitframebuffer-stencil-only.html]: Done

File: sdk/tests/conformance2/rendering/blitframebuffer-stencil-only.html (hunk 2 above)
Comment (null77) [inline sdk/tests/conformance2/rendering/blitframebuffer-stencil-only.html]: Done

File: sdk/tests/conformance2/rendering/blitframebuffer-stencil-only.html (hunk 3 above)
Comment (null77) [inline sdk/tests/conformance2/rendering/blitframebuffer-stencil-only.html]: Done

**Comment by null77** [comment]
Addressed comments, PTAL

**Comment by kenrussell** [comment]
LGTM. Thanks again for this nice test.
